src/main/java/com/kodilla/ecommercee/mapper/UserMapper.java

```diff
@@ -10,7 +10,14 @@
 @Component
 public class UserMapper {
 
-    public User translateToUser(UserDto userDto) {return new User(); }
+    public User translateToUser(final UserDto userDto) {
```

Comment:
Używajmy nazw mapToUser i mapToUserDto, translate się kojarzy z tłumaczeniem na inne języki

Reply:
UserMapper już jest zrobiony wiec to co ja mam po rozwiązaniu konfliktów zniknie